src/utils/removeTagsFromSummary.ts

```diff
@@ -0,0 +1,8 @@
+function removeTagsFromSummary(summary: string): string {
+    let cleanedSummary = summary.split(/<[a-z]+>/);
+    cleanedSummary = cleanedSummary.join("").split(/<\/[a-z]+>/);
+
+    return cleanedSummary.join("");
+}
```

Comment:
Variable shadowing with cleanedSummary. Very hard to keep track of mentally when splitting a string to an array, joining it to a string, splitting it again, and then joining it one more time.